mods/d2k/chrome/dropdowns.yaml

```diff
@@ -26,6 +26,36 @@ ScrollPanel@LABEL_DROPDOWN_TEMPLATE:
 					Width: PARENT_WIDTH - 20
 					Height: 25
 
+ScrollPanel@LABEL_DROPDOWN_WITH_TOOLTIP_TEMPLATE:
```

Comment:
I only just realised that the path tiling tool dropdowns have pointless empty tooltips!

I should fix this. :stuck_out_tongue:

Reply:
You should be able to remove this if #21939 goes through.

Reply:
This widget should be removed now